sdk/storage/azure-storage-queue/azure/storage/queue/_queue_client.py

```diff
@@ -695,7 +695,7 @@ def receive_messages(
     def update_message(
         self, message: Union[str, QueueMessage],
         pop_receipt: Optional[str] = None,
-        content: Optional[Any] = None,
+        content: object = None,
```

Comment:
Left a comment on the API View about this one but I think it should stay as `Any` for consistency. At the very least, I think it needs to stay `Optional`.

Reply:
Doh nice one on dropping the `Optional` from me 😄  But yes I can revert this pending Anna's response

Reply:
As noted, will change to `Optional[object]` and change all other occurences of `content`.

One caveat is that for `QueueMessage` we will leave content as `Any`. While it is more type-safe in most occurences when typing the input, since for `QueueMessage` it's content is typically consumed by the user, I think it would be better to leave this as-is (`Any`) to avoid them from running into type errors and having to have tons of casting operations done with interacting with the fields of their `QueueMessage` they are interacting with.